yarn.lock

```diff
@@ -1601,6 +1601,46 @@
     dequal "^1.0.0"
     isoworker "^0.1.1"
 
+<<<<<<< HEAD
```

Comment:
Is that normal ? `yarn install` doesn't throw any error, so...

Reply:
I believe this is because I added the `react-native-swiper-flatlist` and them I removed it.